plugin-flex-ts-template-v2/src/feature-library/enhanced-crm-container/flex-hooks/components/CRMContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import * as Flex from '@twilio/flex-ui';
+
+import IFrameCRMContainer from '../../custom-components/IFrameCRMContainer'
+
+import { UIAttributes } from 'types/manager/ServiceConfiguration';
+
+const { custom_data } = Flex.Manager.getInstance().serviceConfiguration.ui_attributes as UIAttributes;
+const { enabled } = custom_data.features.enhanced_crm_container;
+
+export function replaceAndSetCustomCRMContainer(flex: typeof Flex, manager: Flex.Manager) {
+
+  if(!enabled) return;
+  
+  const baseUrl = 'https://www.bing.com';
```

Comment:
I think it would be good to put the URL in the configuration at some point. We could parse out task attributes to maintain flexibility, for example `http://mywebsite.com/customer/{{myTaskAttributeNameThatGoesInURL}}/case/{{anotherTaskAttributeName}}`